src/Search/Extensions/ProxyDBExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace SilverStripe\FullTextSearch\Search\Extensions;
+
+use SilverStripe\Core\Extension;
+use TractorCow\ClassProxy\Generators\ProxyGenerator;
+use SilverStripe\FullTextSearch\Search\Updaters\SearchUpdater;
+
+/**
+ * Class ProxyDBExtension
+ * @package SilverStripe\FullTextSearch\Search\Extensions
```

Comment:
Can you please remove these two lines? They don't add any value :)

Reply:
iirc `@package` is a deprecated thing now too, right?